src/connectors/traderjoe/traderjoe.ts

```diff
@@ -111,7 +124,7 @@ export class Traderjoe implements Uniswapish {
    * Default time-to-live for swap transactions, in seconds.
    */
   public get ttl(): number {
-    return this._ttl;
+    return Math.floor(new Date().getTime() / 1000) + this._ttl;
```

Comment:
This is an interface method. We should keep this as true time to live getter, and create an additional helper `deadline` that will return `Math.floor(new Date().getTime() / 1000) + this.ttl`

Reply:
This method is called by and passed to a client of the interface in such a way that mandates `ttl` returns a deadline instead for Uniswapish